src/core/modules/Nebula.hpp

```diff
@@ -125,7 +125,7 @@ friend class NebulaMgr;
 		NebEMO			= 20,  //!< Emission object
 		NebBLL			= 21,  //!< BL Lac object
 		NebBLA			= 22,  //!< Blazar
-		NebMolCld	        = 23,  //!< Molecular Cloud
+		NebMolCld	    = 23,  //!< Molecular Cloud
```

Comment:
at least this looks worse now on Github.

Reply:
So weird, in vscode it looks correct
![Screenshot_20241221_071003](https://github.com/user-attachments/assets/e27e62d6-5b55-47bb-817b-5e6e393d2786)


But in github, no
https://github.com/luzpaz/stellarium/blob/80b9db83a3f3e374ea35b76d15da64baf4e43c2f/src/core/modules/Nebula.hpp#L128

Reply:
It's because tabs shouldn't be used after the first non-whitespace character. They should only be used for indentation (not even alignment!).